app/views/protocols/view_details/_research_involving.html.haml

```diff
@@ -27,7 +27,7 @@
     %h5
       %strong
         = t(:protocols)[:studies][:research_involving][:header]
-        
+
   - if research_types_info.present?
```

Comment:
We could merge this if statement with the one below it (`if research_types_info.human_subjects || research_types_info.vertebrate_animals... etc.`) and use only one `else` block with the no research involving alert to avoid duplicated code.